PCbuild/openssl.vcxproj

```diff
@@ -40,15 +24,36 @@
 
   <Import Project="python.props" />
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
-  
-  <PropertyGroup Label="Configuration">
+
+  <PropertyGroup Label="Configuration" Condition="$(Platform) == 'Win32'">
     <ConfigurationType>Makefile</ConfigurationType>
     <Bitness>32</Bitness>
-    <Bitness Condition="$(Platform) == 'x64'">64</Bitness>
     <ArchName>x86</ArchName>
-    <ArchName Condition="$(Platform) == 'x64'">amd64</ArchName>
     <OpenSSLPlatform>VC-WIN32</OpenSSLPlatform>
-    <OpenSSLPlatform Condition="$(Platform) == 'x64'">VC-WIN64A</OpenSSLPlatform>
+    <SupportSigning>true</SupportSigning>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Configuration" Condition="$(Platform) == 'x64'">
+    <ConfigurationType>Makefile</ConfigurationType>
+    <Bitness>64</Bitness>
+    <ArchName>amd64</ArchName>
+    <OpenSSLPlatform>VC-WIN64A</OpenSSLPlatform>
```

Comment:
appveyor.yml uses VC-WIN64A-masm for this build.  Should this be updated?

Reply:
yes that's what I meant

Reply:
Changing to VC-WIN64A-masm breaks the cpython build with a message that's not helpful:
Error: The operation could not be completed. Unspecified error 

If I'm reading the configure files correctly then "VC-WIN64A" is the 64-bit equivalent of "VC-WIN32 no-asm" and it works with the cpython build.

Reply:
This file shouldn't be being used in the CPython build - it's used to build OpenSSL separately and then we dynamically link to it (I might consider moving it to `Tools` to better reflect this...). Maybe I'll have to try it and see what happens.

Reply:
It's not used in the regular build but I needed to build openssl with the patch the prepare_ssl.bat applies before I could test changing to openssl 1.1.1a.   I assumed that this script is used to build the binaries for cpython-bin-deps based on the comments in PCBuild/readme.txt

Reply:
Actually, this file isn't used at all for the official builds I've been doing ([example](https://dev.azure.com/python/cpython/_build/results?buildId=35360)). And it doesn't quite match how I've been building either... I'll take a look and see if I can get the build done.